addons/website/static/src/snippets/s_countdown/options.js

```diff
@@ -75,7 +81,7 @@ options.registry.countdown = options.Class.extend({
 
         // End Action UI
         this.$el.find('.toggle-edit-message')
-            .toggleClass('d-none', dataset.endAction !== 'message');
+            .toggleClass('d-none', !dataset.endAction.includes('message'));
```

Comment:
Why checking the two values explicitely everywhere else and here checking if it contains "message". Do like everywhere else.